ganga/GangaCore/Lib/Batch/Batch.py

```diff
@@ -169,10 +182,7 @@ def submit(self, jobconfig, master_input_sandbox):
             queue_option = queue_option + " " + self.extraopts
 
         if jobnameopt and job.name != '':
-            # PBS doesn't like names with spaces
-            tmp_name = job.name
-            if isType(self, PBS):
-                tmp_name = tmp_name.replace(" ", "_")
+            tmp_name = self._getLegalJobName()
```

Comment:
this doesn't look right. See #1853

Reply:
Ahh, thanks for finding this. Silly mistake on my part.